docs/make.jl

```diff
@@ -12,5 +12,6 @@ makedocs(
         )
 
 deploydocs(
-    repo = "github.com/JuliaArrays/StaticArrays.jl",
+    repo = "github.com/JuliaArrays/StaticArrays.jl.git",
+    julia = "0.6"
 )
```

Comment:
+`deps = nothing,` because otherwise `deps = Deps.pip("pygments", "mkdocs")` is performed

Reply:
+ `make = nothing,` otherwise `run('mkdocs build')` is performed

Reply:
Done, thanks.